### ISSUE TITLE
Problem on ERROR MinerGetBaseInfo during CRON job

### ISSUE THREAD

**Issue by kelvinyasu** [s0nik42/lotus-farcaster#26]
Hello, 

The CRON was working at the beginning, but it stop working after some issue on my miner restart..
Any idea how I can get this fixed? The dashboard are awesome..

My miner ID are f0400103

```
# TYPE lotus_power gauge
lotus_power { miner_id="f0400103", scope="miner", power_type="RawBytePower" } 0
lotus_power { miner_id="f0400103", scope="miner", power_type="QualityAdjPower" } 0
lotus_power { miner_id="f0400103", scope="network", power_type="RawBytePower" } 3900315704791924736
lotus_power { miner_id="f0400103", scope="network", power_type="QualityAdjPower" } 3901392404504870912
# HELP lotus_power_mining_eligibility return miner mining eligibility
# TYPE lotus_power_mining_eligibility gauge
ERROR MinerGetBaseInfo return no result
KNOWN_REASON your miner reports wrong info to the chain and thats pretty bad (not just for the dashboard)
SOLUTION restart your miner and node
lotus_scrape_execution_succeed { } 0 
```

**Comment by s0nik42** [comment]
Hi,

this come from a bug in lotus. MinerGetBaseInfo return no information when you've got 0 power. It will work again 9 hours after you get your power back to something above 0

**Comment by s0nik42** [comment]
Could you confirm here if that solve your problem ?

**Comment by kelvinyasu** [comment]
Just get back my miner online this morning, but seem the scripts still not working...
I might do a reboot later.. will keep post if its ok

```

root@fc:~/bin# lotus-miner  info
Chain: [sync behind! (12m40s behind)] [basefee 197.292 pFIL]
Miner: f0400103 (32 GiB sectors)
Power: 448 Gi / 3.41 Ei (0.0000%)
        Raw: 448 GiB / 3.404 EiB (0.0000%)
        Committed: 448 GiB
        Proving: 448 GiB
Below minimum power threshold, no blocks will be won
Deals: 2, 128.1 MiB
        Active: 2, 128.1 MiB (Verified: 0, 0 B)

Miner Balance:    4.807 FIL
      PreCommit:  0
      Pledge:     4.286 FIL
      Vesting:    0
      Available:  520.638 mFIL
Market Balance:   70.549 μFIL
       Locked:    70.549 μFIL
       Available: 0
Worker Balance:   7.304 FIL
Total Spendable:  7.824 FIL

Sectors:
        Total: 16
        Proving: 14
        PreCommit2: 1
        Removed: 1
```

**Comment by kelvinyasu** [comment]
Thanks @s0nik42 , it works now after waiting for while.. I didn't do restart, just resumed my miner online and keep it running for while with one pledging job.
